tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/storage-stack.yaml

```diff
@@ -0,0 +1,238 @@
+        DeploymentType: SINGLE_AZ_1
+        ThroughputCapacity: 128
+      SecurityGroupIds:
+        - !Ref FSxSecurityGroup1
+      StorageCapacity: 1024
+      SubnetIds:
+        - !Ref PublicSubnetId
+    Type: 'AWS::FSx::FileSystem'
+  SVMVolume0:
+    Properties:
+      Name: vol0
+      OntapConfiguration:
+        JunctionPath: /vol0
+        SizeInMegabytes: '10240'
+        StorageEfficiencyEnabled: 'true'
+        StorageVirtualMachineId: !Ref StorageVirtualMachineFileSystemResource
+      VolumeType: ONTAP
+    Type: 'AWS::FSx::Volume'
+  StorageVirtualMachineFileSystemResource:
+    Properties:
+      FileSystemId: !Ref FileSystemResource2
+      Name: fsx
+    Type: 'AWS::FSx::StorageVirtualMachine'
+  FSxSecurityGroup2:
+    Properties:
+      GroupDescription: SecurityGroup for testing existing FSx
+      SecurityGroupIngress:
+        - CidrIp: 0.0.0.0/0
+          FromPort: 111
+          IpProtocol: tcp
+          ToPort: 111
+        - CidrIp: 0.0.0.0/0
+          FromPort: 111
+          IpProtocol: udp
+          ToPort: 111
+        - CidrIp: 0.0.0.0/0
+          FromPort: 2049
+          IpProtocol: tcp
+          ToPort: 2049
+        - CidrIp: 0.0.0.0/0
+          FromPort: 2049
+          IpProtocol: udp
+          ToPort: 2049
+        - CidrIp: 0.0.0.0/0
+          FromPort: 20001
+          IpProtocol: tcp
+          ToPort: 20001
+        - CidrIp: 0.0.0.0/0
+          FromPort: 20001
+          IpProtocol: udp
+          ToPort: 20001
+        - CidrIp: 0.0.0.0/0
+          FromPort: 20002
+          IpProtocol: tcp
+          ToPort: 20002
+        - CidrIp: 0.0.0.0/0
+          FromPort: 20002
+          IpProtocol: udp
+          ToPort: 20002
+        - CidrIp: 0.0.0.0/0
+          FromPort: 20003
+          IpProtocol: tcp
+          ToPort: 20003
+        - CidrIp: 0.0.0.0/0
+          FromPort: 20003
+          IpProtocol: udp
+          ToPort: 20003
+      VpcId: !Ref vpc
+    Type: 'AWS::EC2::SecurityGroup'
+  FileSystemResource3:
+    Properties:
+      FileSystemType: OPENZFS
+      OpenZFSConfiguration:
+        DeploymentType: SINGLE_AZ_1
+        RootVolumeConfiguration:
+          NfsExports:
+            - ClientConfigurations:
+                - Clients: '*'
+                  Options:
+                    - rw
+                    - crossmnt
+        ThroughputCapacity: 64
+      SecurityGroupIds:
+        - !Ref FSxSecurityGroup2
+      StorageCapacity: 64
+      SubnetIds:
+        - !Ref PublicSubnetId
+    Type: 'AWS::FSx::FileSystem'
+  OpenZFSVolume0:
+    Properties:
+      Name: vol0
+      OpenZFSConfiguration:
+        NfsExports:
+          - ClientConfigurations:
+              - Clients: '*'
+                Options:
+                  - rw
+                  - crossmnt
+        ParentVolumeId: !GetAtt
+          - FileSystemResource3
+          - RootVolumeId
+      VolumeType: OPENZFS
+    Type: 'AWS::FSx::Volume'
+Outputs:
+  EfsId:
+    Value: !Ref FileSystemResource0
```

Comment:
Minor non blocking comment (I've already added a comment on this, resolved but not addressed).
Outputs are missing a description.

Reply:
same as parameters